automatic/cutepdf/update.ps1

```diff
@@ -0,0 +1,24 @@
+import-module au
+
+$url = 'http://www.cutepdf.com/download/CuteWriter.exe'
+
+function global:au_SearchReplace {
+    @{
+        'tools\ChocolateyInstall.ps1' = @{
+            "(^[$]checksum\s*=\s*)('.*')"   = "`$1'$($Latest.Checksum32)'"
+        }
+     }
+}
+
+function global:au_GetLatest {
+    Write-Host $url
+
+    $temp_file = $env:TEMP + '\CuteWriter.exe'
```

Comment:
The same as before here, without downloading the files if other things are possible.